assets/js/googlesitekit/datastore/user/user-info.test.js

```diff
@@ -108,21 +108,23 @@ describe( 'core/user userInfo', () => {
 				);
 			} );
 		} );
-		describe( 'receiveUserInputState', () => {
-			it( 'requires userInputState param', () => {
+		describe( 'receiveIsUserInputCompleted', () => {
+			it( 'requires isUserInputCompleted param', () => {
 				expect( () => {
-					registry.dispatch( CORE_USER ).receiveUserInputState();
-				} ).toThrow( 'userInputState is required.' );
+					registry
+						.dispatch( CORE_USER )
+						.receiveIsUserInputCompleted();
+				} ).toThrow( 'isUserInputCompleted is required.' );
 			} );
 
 			it( 'receives and sets userInputData', async () => {
```

Comment:
This name reads a bit off. I think it is intended to refer to the name of the state key, which would have been inaccurate before as well 😄

Reply:
Sorry, I was referring to the `userInputData` in the test name which dates back to this commit https://github.com/google/site-kit-wp/commit/05d0bb9d7c34caec368ca90fb6408f30dfe988ed#diff-6659be2b072139854065f96b18a50aeb1905889ee96369146a84d5c560ee472bR88

This made more sense because the action received the given data into `state.userInputState`.

```suggestion
			it( 'receives and sets isUserInputCompleted', async () => {
```